### PR TITLE
Stretch Patches

### PR DESCRIPTION
**Pull request by darkdrgn2k**
Detection should work fine now

### CODE REVIEW

**Comment by darkdrgn2k** [comment]
80211s Workaround and DPKG correction

**Comment by benhylau** [comment]
Are we still waiting on a fix to `status`?
```
scripts/status:    if [ "$(iw dev mesh0 info 2>/dev/null | grep 'type mesh point')" ] && [ "$(iw dev mesh0 info 2>/dev/null | grep 'channel')" ]; then
```

**Comment by darkdrgn2k** [comment]
Re-write the status command to iterate through all interfaces, find the mesh point interfaces and then check if its up.

File: scripts/mesh-point/mesh [hunk 1]
```diff
@@ -36,22 +36,21 @@ fi
...
+# Convert mesh_dev into  802.11s Mesh Point interface
+sudo iw $mesh_dev set type mp
+
 
-# Bring up the mesh0 interface
-sudo ifconfig mesh0 up
+# Bring up the mesh_dev interface
+sudo ifconfig $mesh_dev up
 
 # Optionally assign IPv4 address to the mesh0 interface
```
Comment (benhylau) [inline scripts/mesh-point/mesh]: `s/mesh0/$mesh_dev`

File: scripts/install [hunk 2]
```diff
@@ -12,6 +12,18 @@ if [ -z "$TAG_PROTOTYPE_CJDNS_PI" ]; then
...
+	read -p "Would you like to KILL it to continue (Y/n)? " -n 1 -r
+    echo ""
+    if [[ $REPLY =~ ^[Nn]$ ]]; then
+        echo -e "\e[1;31mPlease stop DPKG Before Proceeding!\e[0m"
+        exit
+    else	
+        echo -e "\e[1;32mKilling DPKG and continuing\e[0m"
+        sudo killall dpkg
+    fi	 
+fi
```
Comment (benhylau) [inline scripts/install]: Newline after this.

File: scripts/mesh-point/mesh [hunk 3]
```diff
@@ -36,22 +36,21 @@ fi
 # Shut down the mesh_dev interface
 sudo ifconfig $mesh_dev down
 
-# Create mesh0 802.11s Mesh Point interface
-if [ -z "$(iw dev | grep mesh0)" ]; then
-	sudo iw phy $mesh_phy interface add mesh0 type mp
-fi
+# Convert mesh_dev into  802.11s Mesh Point interface
```
Comment (benhylau) [inline scripts/mesh-point/mesh]: Delete extra space between `into  802.11s`

File: scripts/mesh-point/mesh [hunk 4]
```diff
@@ -36,22 +36,21 @@ fi
...
+sudo iw $mesh_dev set type mp
+
```
Comment (benhylau) [inline scripts/mesh-point/mesh]: Remove extra newline.

**Pull request merged**